### ISSUE TITLE
feat: AI translated voiceover / Port Voice Over Translation userscript

### ISSUE THREAD

**Issue by lynrayy** [ReVanced/revanced-patches#3280]
### Feature description

ReVanced YouTube app lacks AI voiceover feature which is available using userscripts on desktop.

Implement/port Voice Over Translation userscript which uses Yandex services under the hood in order to translate the voice (in YT, in particular, but not limited to) from one language to another. You can see more details here: https://github.com/ilyhalight/voice-over-translation/blob/master/README-EN.md (contrary to the readme the plugin can actually translate either way, not just to Russian).

We've asked the author @ilyhalight in https://github.com/ilyhalight/voice-over-translation/issues/187 but he doesn't know Kotlin, although he also admits it would be great to have this feature integrated in ReVanced.

### Motivation

Despite English being top-1 international language, there are still many people who don't know English, and also there's a lot of non-English videos. It's considerably more tedious to watch videos in foreign language for people who don't know the language. For instance, I do, but many of my local friends and family members don't know and they would still benefit a lot from that, like my mom who only has her phone and can't use the userscript to translate the video). It would help greately to have this integrated in ReVanced to make videos more accessible worldwide.

### Acknowledgements

- [X] I have checked all open and closed feature requests and this is not a duplicate
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.

**Comment by lynrayy** [comment]
Reopen of https://github.com/ReVanced/revanced-patches/issues/580

Despite the fact that there were no messages in the previous issue for a long time, the feature request is still relevant

**Comment by oSumAtrIX** [comment]
Thanks but the decision in #580 is final for this request.

**Comment by lynrayy** [comment]
There is no decision . No reason why it's closed.
There is only auto-generated reply template. Why it is not planned?

**Comment by oSumAtrIX** [comment]
The request is too complex and therefore we aren't interested in realizing it.
